static/src/javascripts/projects/common/modules/experiments/tests/sign-in-gate-copy-test-variant.js

```diff
@@ -0,0 +1,30 @@
+export const signInGateCopyTestJan2023 = {
```

Comment:
Not massively familar with the client side ab testing framework, is there a reason we need to define this in Frontend if (I'm assuming) it's only used in DCR?